libs/models/schema/ui/cards/cards-module/src/lib/data/sections/highlighted-news.data.ts

```diff
@@ -0,0 +1,14 @@
+import { NewsItem } from "../../..";
+export const __highlightedNews: NewsItem[] = [
+  {
+    title: 'Breaking News 1',
+    description: 'This is a brief description of breaking news 1...',
+    buttonText: 'Read more'
+  },
+  {
+    title: 'Breaking News 2',
+    description: 'This is a brief description of breaking news 2...',
+    buttonText: 'Read more'
+  },
```

Comment:
Right approach, but should have been abstracted properly. Have each item in its right module